environment-setup/tp1/students/students.yml

```diff
@@ -0,0 +1,123 @@
+students:
```

Comment:
Drop all entries from this file except for the `tstudent` one, and rename the file as `students.yml.example`. The idea is that we build this file every term and don't need to commit it.